dmoj/problem.py

```diff
@@ -33,6 +35,9 @@ def __init__(self, problem_id, time_limit, memory_limit, load_pretests_only=Fals
             doc = yaml.safe_load(self.problem_data['init.yml'])
             if not doc:
                 raise InvalidInitException('I find your lack of content disturbing.')
+            logging.getLogger("logging").setLevel(logging.CRITICAL)
```

Comment:
This should be called on `pykwalify.core` instead of the global `logging`, so that not _all_ logs are silenced. It'd also probably be good to not have it be set every time a problem is loaded (just once is enough), so calling it somewhere like [here](https://github.com/DMOJ/judge/blob/master/dmoj/judge.py#L584) should be good.

I think that's about the last thing left with this PR.